annotation-processor/src/main/kotlin/pl/touk/krush/KrushAnnotationProcessor.kt

```diff
@@ -18,11 +18,15 @@ import javax.tools.Diagnostic.Kind.ERROR
 @KotlinPoetMetadataPreview
 @SupportedSourceVersion(SourceVersion.RELEASE_8)
 @SupportedAnnotationTypes("javax.persistence.*")
-@SupportedOptions(KrushAnnotationProcessor.KAPT_KOTLIN_GENERATED_OPTION_NAME)
+@SupportedOptions(
+        KrushAnnotationProcessor.KAPT_KOTLIN_GENERATED_OPTION_NAME,
+        KrushAnnotationProcessor.KRUSH_REFERENCES_OPTION_NAME
+)
 class KrushAnnotationProcessor : AbstractProcessor() {
 
     companion object {
         const val KAPT_KOTLIN_GENERATED_OPTION_NAME = "kapt.kotlin.generated"
+        const val KRUSH_REFERENCES_OPTION_NAME = "krush.references"
```

Comment:
please add (commented) the configuration you mentioned in comment into example/build.gradle